apps/storybook/next.config.js

```diff
@@ -15,7 +15,6 @@ const nextConfig = {
   },
   transpilePackages: [
     "@calcom/app-store",
-    "@calcom/dayjs",
```

Comment:
<img width="489" alt="Screenshot 2023-08-25 at 6 41 05 PM" src="https://github.com/calcom/cal.com/assets/53316345/849d4888-fb07-419f-a516-f8b376886d88">

is this failing the type-check?

Reply:
Not directly, seems also failing when removing type checks

Reply:
Reverted

Reply:
@zomars type checks are still failing for some reason